executor/projection.go

```diff
@@ -366,7 +390,9 @@ func (w *projectionWorker) run(ctx context.Context) {
 			return
 		}
 
+		mSize := output.chk.MemoryUsage()
 		err := w.evaluatorSuit.Run(w.sctx, input.chk, output.chk)
```

Comment:
Do we have a plan to calculate the memory usage inside `evaluatorSuit`? Maybe a new issue?

Reply:
`evaluatorSuit` allocates some temporal buffers when calculating and releases them before return. Do we have to trace these temporal buffers?

Reply:
I think we should, maybe a TODO is enough now. :)

Reply:
A TODO and a new [issue](https://github.com/pingcap/tidb/issues/13999) are added.